recipes/pygtftk/build.sh

```diff
@@ -0,0 +1,3 @@
+ln -s $CC `dirname $CC`/gcc
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
pip install is recommend instead of setup.py usage

Reply:
I have tried using `script: python -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .` in the meta.yaml but it fails for the same reason explained in my first message.
It works if I replace `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`  by `$PYTHON -m pip install --no-deps --ignore-installed --no-cache-dir -vvv .` in build.sh, but I guess this is not the recommended way.